psi4/src/psi4/libmints/mintshelper.cc

```diff
@@ -865,7 +861,13 @@ SharedMatrix MintsHelper::ao_erf_eri(double omega, std::shared_ptr<IntegralFacto
     } else {
         factory = integral_;
     }
-    return ao_helper("AO ERF ERI Integrals", std::shared_ptr<TwoBodyAOInt>(factory->erf_eri(omega)));
+
+    std::vector<std::shared_ptr<TwoBodyAOInt>> ints_vec(nthread_);
+    ints_vec[0] = std::shared_ptr<TwoBodyAOInt>(factory->erf_eri(omega));
+    for (size_t i = 1; i < nthread_; i++) {
+        ints_vec[i] = std::shared_ptr<TwoBodyAOInt>(ints_vec.front()->clone());
+    }
```

Comment:
This might actually be better to run in parallel. That way the memory can get allocated to the processor's local memory.